src/ontology/mappings/mondo_broadmatch_ncit.sssom.tsv

```diff
@@ -0,0 +1,10 @@
+#curie_map:
```

Comment:
let's have a policy of only equivs for ncit

Reply:
Made a ticket here

Reply:
https://github.com/monarch-initiative/mondo/issues/3097